promql/engine.go

```diff
@@ -180,15 +193,13 @@ func (q *query) Exec(ctx context.Context) *Result {
 	var queryIndex int
 	if q.ng.activeQueryTracker != nil {
 		queryIndex = q.ng.activeQueryTracker.Insert(q.q)
+		// Delete query from active log.
```

Comment:
This comment isn't needed now, as all the code is together.

Reply:
done, I also re-scoped queryIndex.